.travis-setup.sh

```diff
@@ -17,42 +17,49 @@
 #  along with this program; if not, write to the Free Software
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-glib_version=2.49.4
+set -e -x
+
+glib_version=2.46.2
```

Comment:
grepping `confiigure`.ac sounds a bit fragile - maybe we need a `versions.txt` file or something that just contains lines like:

```
required_glib_version=2.46
```

So that we could source that into `configure.ac` and `.travis-setup.sh`.